src/PageTypes/DatedUpdateHolder.php

```diff
@@ -139,7 +190,7 @@ public static function ExtractMonths(DataList $updates, $link = null, $currentYe
 		$dates = $updates->dataQuery()
 			->groupby('YEAR("Date")')
 			->groupby('MONTH("Date")')
-			->sort('Date', 'DESC')
+			->sort(DBDate::class, 'DESC')
```

Comment:
Column name, not class name - I'm going to assume the column hasn't been changed to be named `SilverStripe\Model\...` etc.